docs/contributors.rst

```diff
@@ -6,8 +6,24 @@ Contributors
 
 All contributors to the libCellML software project agree to the terms and conditions of the `Apache v2.0 <https://opensource.org/licenses/Apache-2.0>`_ license.
 
-List of Contributors
-====================
+List of Funding Organisations
+=============================
 
  * `University of Auckland <https://www.auckland.ac.nz/>`_
  * `VPR <http://www.virtualrat.org/>`_
+
+List of Contributors
+====================
+
+The following is a list of contributors (in surname alphabetical order) who have contributed lines of source code to the project on or before the 5/3/2019.
+
+* `Robert Blake <https://github.com/rblake-llnl>`_
+* `Michael Clerx <https://github.com/MichaelClerx>`_
+* `Jonathan Cooper <https://github.com/jonc125>`_
+* `Alan Garny <https://github.com/agarny>`_
+* `David Ladd <https://github.com/dladd>`_
+* `Gary Mirams <https://github.com/mirams>`_
+* `David Nickerson <https://github.com/nickerso>`_
+* `Hugh Sorby <https://github.com/hsorby>`_
```

Comment:
What is the point of having that list if we then point to the up-to-date list of contributors? It's bound to be out of date within a few weeks. I would simply have something like:

```
An up-to-date list of contributors can be found at https://github.com/cellml/libcellml/graphs/contributors.
```

Reply:
I like the idea of having the names of contributors available without having to do that extra click.       I am very anti extra mouse clicks.

Reply:
I also slightly prefer being able to see the "current" list immediately and then click through to get all the details. It will take some small effort to keep it from getting too out of date, but manageable...

Reply:
I have created a small Python Script that will generate the list ready for copy and pasting into reStructured text, reducing a little of the burden?

Reply:
People tend to forget to update things that can easily get out of date, which is the reason I prefer the link on its own, but nevermind.